clients/unshaded/src/main/java/tachyon/client/block/TachyonBlockStore.java

```diff
@@ -80,6 +85,28 @@ public BlockInfo getInfo(long blockId) throws IOException {
   }
 
   /**
+   * @return the info of all active block workers
+   * @throws IOException when work info list cannot be obtained from master
+   * @throws TachyonException if network connection failed
+   */
+  public List<BlockWorkerInfo> getBlockWorkerInfoList() throws IOException, TachyonException {
```

Comment:
But it's possible that block worker and file worker do not co-exist on the same machine, no?